lib/app_manifest.rb

```diff
@@ -36,9 +36,9 @@ def canonicalize_env(manifest)
       canonicalize_key(manifest, :env) do |env|
         Hash[
           env.map do |key, value|
-            if value.is_a? String
+            if !value.is_a?(Hash)
               value = {
-                value: value,
+                value: value.to_s,
```

Comment:
@dmathieu To be clear, my assumption here is that nyata won't accept anything but string and hash as valid env values. Reading @raulb's comments on the Trello card seems to imply something else. Does nyata coerce, then validate, or the other way around?

My intent here is that if nyata doesn't coerce this itself, we shouldn't be coercing it here.